Sources/Keyboard/Keyboard.swift

```diff
@@ -10,19 +10,22 @@ public struct Keyboard<Content>: View where Content: View {
     var latching: Bool
     var noteOn: (Pitch) -> Void
     var noteOff: (Pitch) -> Void
+    var rows: Int
```

Comment:
AudioKit style is that plural variables are arrays, not count, so this would be `rowCount`